src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -166,4 +173,31 @@ def consume_captions(caption_list: list) -> dict:
         if 'italics' in caption:
             caption_metadata['italicized_bytes'] = utils.create_bytes_to_italicize_text()
 
+    validate_caption_ids(caption_list)
+
     return caption_metadata
+
+
+def validate_scene_ids(scene_list: list):
+    """Validates the scene IDs to look for duplicate IDs
+
+    :param scene_list:
+    """
+    scene_ids = Counter(scene_list)
+
+    for id,number_of_that_id in scene_ids.items():
+        if number_of_that_id > 1:
+            raise ValueError('There are duplicate scene IDs.')
```

Comment:
if i had 200 scenes and got this error back, how would i go about finding which two scenes are conflicting?